app/views/missions/index.html.erb

```diff
@@ -1 +1,40 @@
-<h1>Missions index</h1>
+<div class='container'>
+  <h2>Home</h2>
+
+  <!-- My organisations -->
+  <% if !current_user.organisations.empty? %>
+    <h4 class="mt-3">My Organisations</h4>
+    <hr class="mb-3">
+    <% current_user.organisations.each do |organisation| %>
```

Comment:
As we saw this mornig, you should include(:organisation) in User.all in your controller